hashicorp/boundary-ui
# feat:  storage policies list view

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12085)

## Description

storage policies list view
**Note**: This list is generated from Mirage, and a couple were created using Postman/API LLB. The `Create` form will be done in the subsequent PR.



## Screenshots (if appropriate):

Using API LLB and Postman:

<img width="1877" alt="Screenshot 2024-01-03 at 3 55 04 PM" src="https://github.com/hashicorp/boundary-ui/assets/15043878/2fff7b45-f64b-46a4-b467-51ac08c02c6b">

Using mirage:
<img width="1824" alt="Screenshot 2024-01-03 at 3 34 25 PM" src="https://github.com/hashicorp/boundary-ui/assets/15043878/74c656e6-036b-4f9f-b2cc-a0892f212db6">


<img width="1888" alt="Screenshot 2024-01-03 at 3 34 06 PM" src="https://github.com/hashicorp/boundary-ui/assets/15043878/6a4387ab-4b6f-4aa7-aa78-c63ff037cfc1">

when there is no policy to list: 
<img width="1532" alt="Screenshot 2024-01-03 at 6 00 19 PM" src="https://github.com/hashicorp/boundary-ui/assets/15043878/9ad198fa-46e6-4d83-af43-5893efa2b5f8">


## How to Test

to test this via boundary controller
- run https://github.com/hashicorp/boundary-enterprise/pull/794
- follow the usual steps to build locally
- create a storage policy in Postman by following the API [guide](https://docs.google.com/document/d/1yR5qkQztWnh8qTjyMXykEHEqJ0PVM3firn-8DYD38MY/edit?pli=1)
- on admin UI, when mirage is skipped, you should see a list of storage policies 

To test via Mirage:
- run admin UI, you should automatically see a list of storage policies in both global and org-level scopes. 




:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
